win32/tests/reals_win32_ut/reals_win32_ut.c

```diff
@@ -65,6 +67,7 @@ TEST_FUNCTION(check_all_c_pal_reals)
     REGISTER_ARITHMETIC_GLOBAL_MOCK_HOOK();
     REGISTER_UUID_GLOBAL_MOCK_HOOK();
     REGISTER_EXECUTION_ENGINE_GLOBAL_MOCK_HOOK();
+    REGISTER_THANDLE_LOG_CONTEXT_HANDLE_GLOBAL_MOCK_HOOK();

```

Comment:
>LOG_CONTEXT_HANDLE_ [](http://example.com/codeflow?start=21&length=19)

I was going to ask why is this here and not in c-logging. Gah. Because no THANDLE in c-logging.

Reply:
This is exactly the reason.